Program.cs

```diff
@@ -335,7 +324,6 @@ private static void ReportCustomersBirthay()
             MenuMain();
             ProcessMain();
         }
-
         private static void ReportProductStock()
         {
             List<Products> objProductsList = new List<Products>();

```

Comment:
You are not using the new instance created.

Reply:
corrected